patroni/dcs/__init__.py

```diff
@@ -898,6 +1281,12 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self.event = Event()
 
     def client_path(self, path: str) -> str:
```

Comment:
Yeah, now the *path* word is very confusing in this context. Originally we used `path` only because of actual hierarchical structures in Etcd v2 and Zookeeper.

It is all about building key names in DCS from the following information:
- namespace (key prefix)
- scope (cluster name)
- key name within the scope

At the end we get something like the following.
1. for etcd, etcd3, and raft:
```
/{namespace}/{scope}/config
/{namespace}/{scope}/leader
/{namespace}/{scope}/members/{my_name}
...
```

2. for consul and zookeeper:
```
{namespace}/{scope}/config
{namespace}/{scope}/leader
{namespace}/{scope}/members/{my_name}
...
```

3. something totally different for K8s (all K8s objects should be created in the same namespace that matches `kuberneted.namespace` parameter):
```
{scope}-leader  # leader configmap
{scope}  # leader endpoint

{scope}-{config}  # config key configmap or endpoint
...
```
Pods (that map to members) may have arbitrary names and identified by their labels.


Now the word `key` would be more appropriate. But we can do it separately.